Lib/Inc/CUartCom.h

```diff
@@ -63,11 +64,11 @@ class CUartCom : public IComChannel
     uint8_t m_status = IDLE;
     CGpioWrapper m_uart_de_pin;
     CFIFOBuffer<char, RX_BUF_SIZE> m_rx_buffer;
-    char m_tx_buffer[TX_BUF_SIZE] = {0};
+    uint8_t m_tx_char;
     uint8_t m_tx_msg_length = 0;
     uint8_t m_rx_char;
     CFIFOBuffer<etl::string<MAX_STRING_SIZE>, MAX_RX_QUEUE_SIZE> m_rx_queue;
-    CFIFOBuffer<etl::string<MAX_STRING_SIZE>, MAX_TX_QUEUE_SIZE> m_tx_queue;
+    CFIFOBuffer<uint8_t, MAX_TX_QUEUE_SIZE> m_tx_queue;
```

Comment:
`MAX_TX_QUEUE_SIZE` name implies there is `MIN_TX_QUEUE_SIZE`. Perhaps a different name such as `TX_BUFFER_SIZE` would reflect the purpose of this variable better.